packages/ui/src/components/header/Header.jsx

```diff
@@ -67,38 +67,45 @@ const Header = ({ openAuthModal }) => {
           />
           <span className={styles["brand-name"]}>{BRANDING.brandName}</span>
         </button>
-        <div className={styles["nav-bar"]}>
-          {NAVBAR_ITEMS.map((item) => {
-            const [itemPath, itemSection] = item.url.split("#");
-            const isActive =
-              item.type === "route"
-                ? currentPath === item.url
-                : currentPath === itemPath && activeSection === itemSection;
+        {!isMobile && (
```

Comment:
We are hiding the navbar in js and in css using `display: none`? Am I getting this right? If yes why such a redundant approach?